packages/icons/src/HarnessIcons.ts

```diff
@@ -1904,6 +1906,8 @@ type HarnessIconName =
   | 'db-devops'
   | 'db-instance'
   | 'db-liquibase'
+  | 'db-preview-inverse'
+  | 'db-preview'
```

Comment:
Using same icon with currentColor is not working in this case, hence added inverse icon also